en-nz/index.html

```diff
@@ -561,7 +561,7 @@ <h3 class="item-title">Not an ICO.</h3>
                 </div>
                 <div class="item-feature__text">
                     <h3 class="item-title">Much faster.</h3>
-                    <p>DigiByte transactions are confirmed in approx. 15 seconds, which is 40 times faster than Bitcoin and 10 times faster than Litecoin. <a href="https://wikipedia.org/wiki/SegWit" target="_blank">SegWit</a> enables up to <a href="https://www.dgbwiki.com/index.php?title=DigiSpeed" target="_blank">1066</a> on-chain transactions per second and negligible fees make DigiByte the ideal currency for daily payments.
+                    <p>DigiByte transactions are confirmed in approx. 15 seconds, which is 40 times faster than Bitcoin and 10 times faster than Litecoin. <a href="https://wikipedia.org/wiki/SegWit" target="_blank">SegWit</a> enables up to <a href="https://www.dgbwiki.com/index.php?title=DigiSpeed" target="_blank">4 times</a> amount of transactions per second and negligible fees make DigiByte the ideal currency for daily payments.
```

Comment:
Would be nice if this (as well as all other english versions of course) said "...make DigiByte the ideal currency for quick daly paymens"

Reply:
added "quick" word
see commit e838b1ae7038367eebf5cc6b1bc41c45937daa57